src/pages/gutenberg/free-audio-books.vue

```diff
@@ -217,21 +262,44 @@ export default {
           content: this.$t('gutenbergFreeAudioBooksPage_og_description'),
         },
       ],
+      script: scripts,
     };
   },
-  async mounted() {
-    await this.loadCSVFile(DISPLAY_COLUMN);
+  computed: {
+    filteredData() {
+      if (!this.searchKeyword) {
+        return this.csvData;
+      }
+      const lowerCaseKeyword = this.searchKeyword.toLowerCase();
+      const filtered = this.csvData.filter(row =>
+        row.classTitle
+          .toString()
+          .toLowerCase()
+          .includes(lowerCaseKeyword)
```

Comment:
Please also allow searching for author

Reply:
[🚸 Enable to search author](https://github.com/likecoin/liker-land/pull/1620/commits/5a420c6125fbae82208d363577c0fca56fb5a0cf)